src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/Inputs/SARIF1004.ExpressUriBaseIdsCorrectly_Invalid.sarif

```diff
@@ -10,13 +10,13 @@
         }
       },
       "originalUriBaseIds": {
-        "PROJECT_ROOT": {
+        "REPOROOT": {
           "uri": "project",
           "description": {
             "text": "This artifactLocation has no uriBaseId, so its uri, if present, must be absolute. But it isn't. It also doesn't end with a slash."
           }
         },
-        "SOURCE_ROOT": {
+        "SRCROOT": {
           "uri": "src",
           "uriBaseId": "PROJECT_ROOT",
```

Comment:
>"PROJECT_ROOT [](start = 23, length = 13)

PROJECT_ROOT was renamed, yes?

Reply:
since your first comment about the terms, i will rollback to `source_root` and wait for larry's comment, ok?

---
In reply to: [445586890](https://github.com/microsoft/sarif-sdk/pull/1938#discussion_r445586890) [](ancestors = 445586890)

Reply:
It's fine to bring this change into conformance with Larry's original proposal. To be consistent, we would rename this to 'REPOROOT' am I correct?

---
In reply to: [445589112](https://github.com/microsoft/sarif-sdk/pull/1938#discussion_r445589112) [](ancestors = 445589112,445586890)

Reply:
yes, it would. i will check with him, because, we have many keys that would alert in this scenario and if that is expected. 

---
In reply to: [445594014](https://github.com/microsoft/sarif-sdk/pull/1938#discussion_r445594014) [](ancestors = 445594014,445589112,445586890)

Reply:
I rollback to the initial version and enabled the verbose by passing true as parameter. What do you think?

---
In reply to: [445595450](https://github.com/microsoft/sarif-sdk/pull/1938#discussion_r445595450) [](ancestors = 445595450,445594014,445589112,445586890)